Lib/zoneinfo/_common.py

```diff
@@ -22,6 +22,23 @@ def load_tzdata(key):
         # UnicodeEncodeError: If package_name or resource_name are not UTF-8,
         #   such as keys containing a surrogate character.
         raise ZoneInfoNotFoundError(f"No time zone found with key {key}")
+    except OSError as exc:
```

Comment:
Why not using (IsADirectoryError, PermissionError)? exc doesn't seem to be used.